primitives/src/vid/advz.rs

```diff
@@ -820,16 +993,24 @@ mod tests {
         let (payload_chunk_size, num_storage_nodes) = (256, 512);
         let mut rng = jf_utils::test_rng();
         let srs = init_srs(payload_chunk_size, &mut rng);
-        let advz =
-            Advz::<Bls12_381, Sha256>::new(payload_chunk_size, num_storage_nodes, 1, srs).unwrap();
-        let payload_random = init_random_payload(1 << 20, &mut rng);
+        let mut advz =
+            Advz::<Bn254, Sha256>::new(payload_chunk_size, num_storage_nodes, 1, &srs).unwrap();
+        #[cfg(feature = "gpu-vid")]
+        let mut advz_gpu =
+            AdvzGPU::<'_, Bn254, Sha256>::new(payload_chunk_size, num_storage_nodes, 1, &srs)
+                .unwrap();
+
+        let payload_random = init_random_payload(1 << 25, &mut rng);
+
+        #[cfg(feature = "gpu-vid")]
+        let _ = advz_gpu.commit_only(payload_random.clone());
 
         let _ = advz.commit_only(payload_random);
     }
 
     #[test]
     fn sad_path_verify_share_corrupt_share() {
-        let (advz, bytes_random) = avdz_init();
+        let (mut advz, bytes_random) = avdz_init();
```

Comment:
Typo here, avdz -> advz

Reply:
fixed in [991d4a9](https://github.com/EspressoSystems/jellyfish/pull/522/commits/991d4a9bedfb039e94379d6c7002b2d193fbd455)